rosa_release_notes/rosa-release-notes.adoc

```diff
@@ -0,0 +1,104 @@
+
+* **Upgrade handling**: The commands `rosa upgrade cluster` and `rosa upgrade account` roles are seamless and backward compatible with all the older ROSA versions and older ROSA clusters. When you run these commands, the old ROSA clusters and their relevant STS roles and policies are updated to the latest versions.
+
+* **New AWS roles supporting the ROSA UI**: The tools to manage the `ocm-role` and `user-role` and use the ROSA provisioning UI are now available in the ROSA CLI as of version 1.2.0. The `ocm-role` and `user-role` are also instrumental in facilitating ROSA STS upgrades done from the {cluster-manager-first} UI only.
+
+[id="rosa-documentation-enhancements"]
+=== Documentation enhancements
+
+The following list describes the key documentation features of ROSA:
+
+* **Getting started with ROSA**: ROSA 4.10 now includes a getting started guide. Getting Started with ROSA defines basic terminology and provides role-based next steps for developers and administrators. The tutorials walk new users through the web console and the OpenShift CLI interfaces. New users can accomplish the following tasks through the Getting Started:
+  ** Setting up the environment
+  ** Creating a ROSA cluster with STS using the default options
+  ** Creating a cluster administrator user for quick cluster access
+  ** Configuring an identity provider and granting cluster access
+  ** Accessing a cluster through the web console
+  ** Deploying an application from the Developer Catalog
+  ** Revoking administrator privileges and user access
+  ** Deleting a ROSA cluster and the AWS STS resources
+
+For more information, see xref:../rosa_getting_started/rosa-getting-started.adoc#rosa-getting-started[Getting started with {product-title}].
+
+* **Documentation restructure**: The ROSA product documentation went through two phases of restructuring focused on making the following more intuitive, clear, and findable:
+  ** Navigation menu
+  ** Book content
+  ** Topic flow
+
+In addition, redirects were put in place to help ensure a seamless navigation experience in the ROSA documentation.
+
+[id="technical-changes_{context}"]
+== Notable technical changes
+ROSA 4.10 introduces the following notable technical changes.
+
+[id="notable-changes-ephemeral-volumes"]
+=== Encrypted ephemeral instance volumes
+Clusters built in July 2022 and later use encrypted ephemeral instances as part of the pre-create cluster workflow. These will pass Service control policies (SCPs) that demand encrypted instance volumes only.
+
+[id="notable-changes-throttling-improvements"]
+=== Throttling improvements for ROSA CLI
+Delays, intervals and timeouts were improved to reduce the possibility of being throttled when addressing the AWS APIs.
```

Comment:
Can you explain here what was being throttled maybe? It's not clear to me from reading this sentence.
e.g. "Delays, intervals and timeouts were improved to reduce the possibility of throttling for X when addressing the AWS APIs."

Reply:
Added the info.